app/graphql/mutations/update_car_monthly_mileage.rb

```diff
@@ -3,17 +3,12 @@ class UpdateCarMonthlyMileage < Mutations::BaseMutation
     description "Update monthly milagemand footprint for car"
 
     argument :total_mileage, Integer, required: true
-    argument :car_id, Integer, required: true
-    argument :month, String, required: true
-    argument :year, String, required: true
+    argument :car_monthly_mileage_id, Integer, required: true
```

Comment:
Looks good! We might want to change this from `Integer` to `ID`, but I think that's just a matter of how we expect the data to be parsed by GraphQL